cupy/_util.pyx

```diff
@@ -26,6 +26,10 @@ CUDA_ARRAY_INTERFACE_EXPORT_VERSION = int(
     os.environ.get('CUPY_CUDA_ARRAY_INTERFACE_EXPORT_VERSION', 3))
 
 
+CUPY_DLPACK_EXPORT_VERSION = float(
```

Comment:
How about using `60` here, like `DLPACK_VERSION`? Using float can be problematic when comparison.

Reply:
`DLPACK_VERSION` is very bad. Before v0.6, it is `050` for v0.5 and so on. The comparison would still work if users type it correctly, but I think they should not have to worry about how exactly `DLPACK_VERSION` was formatted.

Reply:
(See the removed stringify code in `cupy/_core/dlpack.pyx`.)

Reply:
I think @kmaehashi mostly meant to use `CUPY_DLPACK_EXPORT_VERSION=60` or `CUPY_DLPACK_EXPORT_VERSION=50`, just to avoid the float conversion, we decide the version values instead of using `DLPACK_VERSION` directly.

I agree with him, we should avoid `float` unless there is a strong and very good reason for it.

Reply:
I mean why would users know `60` or `50` are acceptable inputs, instead of say `050`? And how about other values? I think the version string is implementation detail and users, if interpreting this env var literally, would want to set it according to [the DLPack public releases](https://github.com/dmlc/dlpack/releases), no?